gui/MainWindow.cxx

```diff
@@ -3383,6 +3400,52 @@ void MainWindow::applySimilarityTransform()
   d->project->write();
 }
 
+//-----------------------------------------------------------------------------
+void MainWindow::computeCamera()
+{
+  QTE_D();
+
+  // Get inputs
+  auto features = d->groundControlPointsHelper->registrationTracks();
+  auto landmarks = d->groundControlPointsHelper->registrationLandmarks();
+
+  // Set up algorithm
+  auto config = kv::config_block::empty_config();
+  config->set_value("algorithm:type", "ocv");
```

Comment:
Fine for this PR but we should change this later to read the config from an installed config file.  This will allow users to select and configure other algorithms.